mantidproject/mantid
# Fix FindSXPeaks returning incorrect TOF

This fixes the issue with FindSXPeaks return the incorrect time of flight.

**To test:**

Here's a script that will reproduce the issue:

``` python
ws = LoadRaw(Filename='ENGINX00252486.raw')
ws = CropWorkspace(ws, StartWorkspaceIndex=2269, EndWorkspaceIndex=2270)
test = FindSXPeaks(InputWorkspace=ws, OutputWorkspace='test')
```

The peaks found in the workspace test will not match the TOF. Apply the change the correct TOF should then be found.

Fixes #17331 .

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
